recipes/allennlp-semparse/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: allennlp-semparse
+  version: {{ version }}
+
+source:
+  url: https://github.com/allenai/allennlp-semparse/archive/v{{ version }}.tar.gz
+  sha256: ffe539df692f4ae90d16a4b690b8203e596d0e2db71b8ef79dbf3614bbbcdc08
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```

How about noarch? It's fine for the staged-recipes CI to fail with missing dependencies on other platforms
```suggestion
  noarch: python
```